supervisor/process.py

```diff
@@ -209,6 +211,9 @@ def spawn(self):
         self._assertInState(ProcessStates.EXITED, ProcessStates.FATAL,
                             ProcessStates.BACKOFF, ProcessStates.STOPPED)
 
+        if delaystart:
+            time.sleep(self.config.startdelay)
```

Comment:
You can control the individual processes with `supervisorctl`.

With this program config:

    [program:foo]
    command=/bin/foo
    process_name=foo_%(process_num)s
    numprocs=2

Start all processes in the `foo` group:

    $ supervisorctl start foo:

Start individual `foo` processes:

    $ supervisorctl start foo:foo_0
    $ supervisorctl start foo:foo_1